.travis.yml

```diff
@@ -27,6 +27,15 @@ matrix:
       env:
         - LANG=js TEST_SUITE=test
 
+    - os: linux
+      dist: trusty
+      sudo: required
+      language: node_js
+      node_js:
+        - "8.6.0"
```

Comment:
Travis installs 8.9 something when using 8. Does it really needs 8.6.0 specifically?